apps-rendering/src/relatedContent.ts

```diff
@@ -68,6 +69,14 @@ const parseHeaderImage = (content: Content): Image | undefined => {
 	}
 };
 
+const getBylineImage = (content: Content): string | undefined => {
+	const contributors = content.tags.filter(tag => tag.type === TagType.CONTRIBUTOR);
+
+	if (contributors.length > 0) {
+		return contributors[0].bylineLargeImageUrl
+	}
+};
```

Comment:
There are a couple of functions already available for retrieving contributor information from `Content` objects that may be useful:

- `articleContributors` in `capi.ts`
- `parseContributors` in `contributors.ts`